generative_ai/batch_predict/batch_gemini_predict.py

```diff
@@ -0,0 +1,91 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from google.cloud.aiplatform import BatchPredictionJob
+
+
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from google.cloud.aiplatform import BatchPredictionJob
+
+import os
+
+
+PROJECT_ID = os.getenv("GOOGLE_CLOUD_PROJECT")
+# LOCATION = "us-central1"
+
+
+def batch_prediction_gemini_example(
+    input_uri: str = None, output_uri: str = None
```

Comment:
```suggestion
    input_uri: Optional[str] = None, output_uri: Optional[str] = None
```

Also add:

```py
from typing import Optional
```

Reply:
updated.

Reply:
I think this is the same as it was before. Also, should these parameters be optional/default None? Aren't those required parameters for the method?

Reply:
This is an old version

Reply:
updated params